### ISSUE TITLE
openrecoveryscript: how to deal with system-image and vendor-image partitions?

### ISSUE THREAD

**Issue by MisterFloppy** [TeamWin/Team-Win-Recovery-Project#1521]
- [x] I am running an official build of TWRP, downloaded from https://twrp.me/Devices/
- [x] I am running the latest version of TWRP
- [x] I have read the FAQ (https://twrp.me/FAQ/)
- [x] I have searched for my issue and it does not already exist

**Device codename**: 
Pocophone F1
**TWRP version**: 
twrp-3.3.0-0-beryllium

#### WHAT STEPS WILL REPRODUCE THE PROBLEM?
Trying to backup system-image and vendor-image partitions via openrecoveryscript by using "backup" command and "set tw_backup_list" parameter.

#### WHAT IS THE EXPECTED RESULT?
System-image and vendor-image partitions should be possible to be included in the backup process initiated through the openrecoveryscript.
#### WHAT HAPPENS INSTEAD?
System-image and vendor-image partitions are not possible to be included in the backup process initiated through the openrecoveryscript.

Additional information:
I would like to initiate a NAND-backup through an openrecoveryscript regarding the following partitions:
- boot
- cache
- data
- recovery
- system-image
- vendor-image

For the corresponding command line operation to specify the firat 4 partitions to be included in the backup process I would write "backup BCDR", but I'm totally clueless how to deal with the system-image and vendor-image partitions, obviously, there are no characters representing these ones...

Selecting all the mentioned partitions (including the system-image and vendor-image partitions!) directly in the TWRP recovery menu is no problem at all, which produces the following entries in the config file ".twrps" under Android:
tw_backup_list:/boot;/cache;/recovery;/system_image;/vendor_image;/data

I've tried to set the tw_backup_list parameter in my openrecoveryscript corresponding to the entries listed in the config file ".twrps", but since the "backup" command requires the enumeration of the partitions to be backed up, too, I can't see a way to "override" the lacking of the characters representing the system-image and vendor-image partitions... trying to do so results in not backing up any partition at all...

Is there a way to solve this problem? If not, could someone please adjust TWRP so that system-image and vendor-image partitions are selectable via openrecoveryscript, too?

Thank you in advance!

**Comment by RuggedHunterMods** [comment]
Yes..... exactly what he said.  We need variables assigned to accommodate openrecoveryscript command switch parameters which refer to system_image and vendor_image.  Alternative solutions might be a method of running a backup thru openrecoveryscript which uses the currently selected partitions and settings in TWRP recovery.

**Comment by PixelShober** [comment]
Is there a solution yet.. ? :/
